x-pack/plugins/ml/public/application/datavisualizer/index_based/components/field_data_card/content_types/date_content.tsx

```diff
@@ -4,64 +4,75 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import React, { FC } from 'react';
-import { EuiIcon, EuiSpacer, EuiText } from '@elastic/eui';
+import React, { FC, ReactNode } from 'react';
+import { EuiBasicTable, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
 // @ts-ignore
 import { formatDate } from '@elastic/eui/lib/services/format';
-
 import { FormattedMessage } from '@kbn/i18n/react';
 
+import { i18n } from '@kbn/i18n';
 import { FieldDataCardProps } from '../field_data_card';
-import { roundToDecimalPlace } from '../../../../../formatters/round_to_decimal_place';
-
+import { ExpandedRowFieldHeader } from '../../../../stats_datagrid/components/expanded_row_field_header';
 const TIME_FORMAT = 'MMM D YYYY, HH:mm:ss.SSS';
+interface SummaryTableItem {
+  function: string;
+  display: ReactNode;
+  value: number | string | undefined | null;
+}
 
 export const DateContent: FC<FieldDataCardProps> = ({ config }) => {
   const { stats } = config;
+  if (stats === undefined) return null;
 
-  const { count, sampleCount, earliest, latest } = stats;
-  const docsPercent = roundToDecimalPlace((count / sampleCount) * 100);
-
-  return (
-    <div className="mlFieldDataCard__stats">
-      <div>
-        <EuiText size="xs" color="subdued" data-test-subj="mlFieldDataCardDocCount">
-          <EuiIcon type="document" />
-          &nbsp;
-          <FormattedMessage
-            id="xpack.ml.fieldDataCard.cardDate.documentsCountDescription"
-            defaultMessage="{count, plural, zero {# document} one {# document} other {# documents}} ({docsPercent}%)"
-            values={{
-              count,
-              docsPercent,
-            }}
-          />
-        </EuiText>
-      </div>
-
-      <EuiSpacer size="m" />
+  const { earliest, latest } = stats;
 
-      <div data-test-subj="mlFieldDataCardEarliest">
+  const summaryTableTitle = i18n.translate('xpack.ml.fieldDataCard.cardNumber.summaryTableTitle', {
+    defaultMessage: 'summary',
```

Comment:
There seems to be a few spots where these aren't Title Case.

Reply:
Updated here https://github.com/elastic/kibana/pull/85726/commits/20ad0dbab26023ed3b5b9c1635af8423002d908d